RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+# Version 0.9.4
```

Comment:
You will need to adjust this via rebase due to the changes I made on 14609 and just merged into the baseline.